tests/console/sshd.pm

```diff
@@ -74,7 +74,7 @@ sub run {
 
     # Generate RSA key for SSH and SCP it to our new user's profile
```

Comment:
Please make sure that the comment reflects that.

Reply:
Fixed.